### ISSUE TITLE
Malfunction of deselecting the selected text using mouse click

### ISSUE THREAD

**Issue by wocala** [philc/vimium#2527]
When you want to deselect the selected text in a web page, you can use mouse click. But Sometimes vimium may "block" the click and you need to click twice or three times even more. I've tried to disable and renable extensions and found out it is vimium that caused this problem. 
Let me know if more infromation is needed. Thanks.

**Comment by smblott-github** [comment]
Hi @wocala.  I can't reproduce this.  Also, at first cut it seems unlikely to be a Vimium issue since Vimium doesn't really do anything with mouse events.

**Comment by wocala** [comment]
Hi, @smblott-github 
Can you try this website? use "ctrl+a" to select all 
http://www.economist.com/

**Comment by wocala** [comment]
please try twice or more. @smblott-github

**Comment by smblott-github** [comment]
> Can you try this website?

Yes.  Select all, then click anywhere to deselect.  Works fine.  I tried many (e.g. 10+) times.

So, still cannot reproduce.

**Comment by wocala** [comment]
@smblott-github 
That's very wired. Looks like something happend only to me. Do you have any suggestions? Thanks.

**Comment by Cypher1** [comment]
Also having this issue. Any progress?
